app/src/main/java/com/example/fittracker/activities/UserSettingsActivity.java

```diff
@@ -0,0 +1,39 @@
+package com.example.fittracker.activities;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.example.fittracker.R;
+
+import butterknife.ButterKnife;
+import butterknife.OnLongClick;
+
+public class UserSettingsActivity extends AppCompatActivity {
```

Comment:
where is the Model, View, and Presenter of this activity? all methods must call the presenter here

Reply:
Done